xnat_tools/dicom_export.py

```diff
@@ -217,19 +147,12 @@ def main(args):
         export_session_dir,
     )
 
+    # Close connection(I don't think this works)
     connection.delete(f"{host}/data/JSESSION")
```

Comment:
This does not work. By calling `connection.delete` you're actually creating a new session to delete the old session

Reply:
Okay, I'll remove the line